### PR TITLE
update to bottom button

### PR DESCRIPTION
**Pull request by The-Jewell**
#1 created new dart file in lib/screens/ component: bottom_buttons
#2 refactored bottom_buttons code to look more clean, added push route to home button
#3 removed bottom button code from main_menu_screen and just called the bottom button class to show it on the main menu screen

### CODE REVIEW

File: lib/Screens/Components/bottom_buttons.dart [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.blue,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          Expanded(
+            child: Container(
+              color: Colors.blue[700],
```
Comment (Tomentosa) [inline lib/Screens/Components/bottom_buttons.dart:17]: Why Colors.blue[700] vs  Colors.blue like the others?

Looks like its bein pressed accidentally

**Comment by The-Jewell** [comment]
It is to show “selected” — still need to create the logic that will make
the active tab have that color, but the default screen will be the home so
set it’s color to that initially.

File: lib/Screens/Components/bottom_buttons.dart [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                          children: [
+                            FaIcon(FontAwesomeIcons.home),
+                            Text('MENU'),
+                          ],
+                        ),
+                        onPressed: () {
+                          Navigator.push(
+                            context,
+                            MaterialPageRoute(
+                              builder: (context) => MainMenuScreen(),
```
Comment (Tomentosa) [inline lib/Screens/Components/bottom_buttons.dart:39]: It seems to create another instance of the MainMenuScreen() object when onPressed: () is executed. I tested this by pressing Menu a bunch and then there were multiple "back" navigation arrows in the top left hand corner

**Comment by The-Jewell** [comment]
Yeah,

I need to create the logic that checks the current route and then doesn’t
open a new screen if it is on that screen already.  It’s initial stages and
definitely not done was just super late so updated what I had so everyone
could see

**Comment by Tomentosa** [comment]
What is your plan for copying the UI mockup/should we wait to merge until it's done/matching?
image

**Comment by The-Jewell** [comment]
Ah, I thought I had it matching the mock-up but was going off the TDD
images which had the top bar with the profile picture and title cropped
out. I will have to go back and fix this.

**Review by Tomentosa — CHANGES_REQUESTED** [review]
Tested on Galaxy Tab A: Android

TODO: Screen check logic to prevent many windows on re-press of each button
TODO: Logic to adjust menus to the colors of the active windows to match the UI/UX mockup
TODO: mimic layout of UI/UX Mockup Video [https://umgcdev361.sharepoint.com/✌️/r/sites/SWEN670Spring2022/Shared%20Documents/General/UX%20Stuff/MemorEz_MockUp_PaitentMode.mp4?csf=1&web=1&e=uzgwbP](https://umgcdev361.sharepoint.com/:v:/r/sites/SWEN670Spring2022/Shared%20Documents/General/UX%20Stuff/MemorEz_MockUp_PaitentMode.mp4?csf=1&web=1&e=uzgwbP)

**Pull request closed without merging**